tests/nnfw_api/src/GenModelTest.h

```diff
@@ -162,10 +236,9 @@ class GenModelTest : public ::testing::Test
         nnfw_tensorinfo ti;
         NNFW_ENSURE_SUCCESS(nnfw_input_tensorinfo(_so.session, ind, &ti));
         uint64_t input_elements = num_elems(&ti);
-        _so.inputs[ind].resize(input_elements);
-
+        _so.inputs[ind].resize(input_elements * sizeOfNnfwType(ti.dtype));
         ASSERT_EQ(nnfw_set_input(_so.session, ind, ti.dtype, _so.inputs[ind].data(),
-                                 sizeof(float) * input_elements),
+                                 _so.inputs[ind].size()),
```

Comment:
If size() is 0, data() may or may not return a null pointer. So, is there no problem?

Reply:
Good catch, will fix!